src/vdc_collection.rs

```diff
@@ -160,6 +160,9 @@ mod tests {
     async fn test_vdc() {
         let smi: Arc<dyn StorageManagerInterface> = Arc::new(LocalStore);
         let vdc = VdcCollection::new(smi);
+        for id in vdc.all_entries().await.unwrap() {
+            vdc.delete(id).await.unwrap();
+        }
```

Comment:
I wonder if we should use a `MemoryStore` rather than `LocalStore` for unit tests to avoid this problem entirely?